vagrant-post-script/install_other_utils.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "[Vagrant] ###################     Installing nvm"
+curl -sS -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+
+
+echo "[Vagrant] ###################     Installing make because apparently it's not standard no more??"
```

Comment:
make is indeed not standard anymore in Ubuntu 20.04. Shall we change the image in Vagrantfile also to Ubuntu 20.04 then?